### ISSUE TITLE
Neovim, dispatch and command prompt

### ISSUE THREAD

**Issue by aliev** [radenling/vim-dispatch-neovim#2]
Hi, thank you very much for a great extension!

the most critical issue that I noticed is the lack of support for terminal input. for example, I run Django tests, and I have the following command output:

```
|| Creating test database for alias 'default'...
|| 
|| Got an error creating the test database: database "test_infty" already exists
|| 
|| 
|| Type 'yes' if you would like to try deleting the test database 'test_dbname', or 'no' to cancel: 
```

(from :Copen)

Is it possible to make so that the tests were run in neovim terminal? and "noiselessly" if I run `Make` command with "bang"? `:Make!` and `:Make`

Thanks!

**Comment by radenling** [comment]
Yes, that should be possible. I guess mimicking how vim-dispatch works when running in tmux would be a good idea. Then `Make` would open a terminal in a split which allows for input and `Make!` would run in the background. Would that work for you?

**Comment by aliev** [comment]
@radenling I mean without tmux, by using neovim terminal :)

**Comment by radenling** [comment]
Yeah, I just used how vim + tmux + dispatch works as a suggestion of how this plugin should work in neovim. I've experimented with it a bit on branch [foreground-terminal](/radenling/vim-dispatch-neovim/tree/foreground-terminal). It might need some way to configure the window height and position of the terminal, but other than that, it seems to work. Feel free to have a look and tell me what you think.
